.ci/pitest.sh

```diff
@@ -152,7 +152,13 @@ pitest-javadoc)
 #   ;;
 
 *)
-  echo "Unexpected argument: $1"
+  if [[ -n "$1" && "--help" != "$1" ]]; then
```

Comment:
Not familiar with bash but can we add `--help` to the switch and have it drop through the `*)` so it is clear it is an option?
Something like:
````
--help)
*)
  ...execution...
````

Reply:
`--help|*)`